core/src/components/searchbar/readme.md

```diff
@@ -4,6 +4,15 @@ Searchbars represent a text field that can be used to search through a collectio
 
 A Searchbar should be used instead of an input to search lists. A clear button is displayed upon entering input in the searchbar's text field. Clicking on the clear button will erase the text field and the input will remain focused. A cancel button can be enabled which will clear the input and lose the focus upon click.
 
+### Keyboard Display
+
+#### Android
+
+By default, tapping `ion-searchbar` will cause the keyboard to appear with a magnifying glass icon on the submit button. You can optionally set the `inputmode` property to `"search"`, which will change the icon from a magnifying glass to a carriage return.
```

Comment:
```suggestion
By default, tapping the input will cause the keyboard to appear with a magnifying glass icon on the submit button. You can optionally set the `inputmode` property to `"search"`, which will change the icon from a magnifying glass to a carriage return.
```

Sorry, last change, I think it makes more sense to talk about tapping the input itself since the searchbar can include the side buttons (cancel, clear)